FreeRTOS/FreeRTOS
# Update libslirp network interface API and +TCP submodule pointer

Update libslirp network interface API and +TCP submodule pointer

Description
-----------

Update libslirp network interface API for coding convension of other ports.

Test Steps
-----------

- Pass POSIX demo

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
